buster/formatters/prompts.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from dataclasses import dataclass
+
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+@dataclass
+class SystemPromptFormatter:
+    text_before_docs: str = ""
+    text_after_docs: str = ""
+    max_words: int = 4000
+
+    def format_documents(self, matched_documents: pd.DataFrame, max_words: int) -> str:
+        # gather the documents in one large plaintext variable
+        documents_list = matched_documents.content.to_list()
+        documents_str = ""
+        for idx, doc in enumerate(documents_list):
+            documents_str += f"<DOCUMENT> {doc} <\DOCUMENT>"
```

Comment:
I think you need to escape the \ in <\DOCUMENT>, it's creating a warning

Reply:
fixed